forms-flow-theme/scss/_button.scss

```diff
@@ -22,4 +22,112 @@
 
 .btn-link:hover {
     text-decoration: underline !important;
+}
+
+.btn-primary { 
+    background-color: var(--button-primary-background);
```

Comment:
without over riding the class can we use direct variable over ride @Bonymol-aot @Josephalexantony-aot @fahad-aot

Reply:
@shuhaib-aot there is no variable like $btn-primary-bg in bootstrap. still in this case as we already override the primary, that should reflect here as well.

Reply:
Updated.